### PR TITLE
Truncate image title to max 2000 chars

### PR DESCRIPTION
**Pull request by sazap10**

### CODE REVIEW

File: types.go [hunk 1]
```diff
@@ -56,3 +56,17 @@ type PlotExpr struct {
 func (expr PlotExpr) String() string {
```
Comment (foygl) [inline types.go:56]: Do we actually still use this, or should we just replace it with the code below and not have a separate "TruncatedString" function?

File: types.go [hunk 2]
```diff
@@ -56,3 +56,17 @@ type PlotExpr struct {
...
 }
+
+func (expr PlotExpr) TruncatedString() string {
+	formulaString := expr.Formula
+	levelString := fmt.Sprintf("%.2f", expr.Level)
+	formulaLength := len(expr.Formula)
+	operatorLength := len(expr.Operator)
+	levelLength := len(levelString)
+	truncateText := "[TRUNCATED]"
+	if formulaLength + operatorLength + levelLength > MAX_TEXT_LENGTH {
```
Comment (foygl) [inline types.go]: This doesn't account for the spaces, does it?

File: types.go (hunk 1 above)
Comment (sazap10) [reply types.go:56]: it is used https://github.com/bugsnag/promalert/blob/7d9beb1d6d2172d2dc998aa0effa152c75748132/plotter.go#L32 which is why i didnt replace it

File: types.go (hunk 2 above)
Comment (sazap10) [reply types.go]: no it doesnt

**Review by foygl — APPROVED** [review]

**Pull request merged**